sql-plugin/src/main/spark320/scala/org/apache/spark/sql/rapids/arithmetic.scala

```diff
@@ -54,11 +54,23 @@ case class GpuIntegralDivide(
     right: Expression,
     failOnError: Boolean = SQLConf.get.ansiEnabled) extends GpuIntegralDivideParent(left, right)
 
+object GpuDecimalDivide {
+  def apply(left: Expression, right: Expression, dataType: DecimalType): GpuDecimalDivide = {
+    val ansi = SQLConf.get.ansiEnabled
```

Comment:
Just for my understanding - The plan is to move away from using the ANSI config https://github.com/NVIDIA/spark-rapids/issues/13144, correct? . Will this be removed later on while working on 13144.

Reply:
Yes I am working on that now. It is a lot of work and I didn't want to make too many changes in a single PR. So I am going to keep this here for now, because it matches the other code, and then in a separate PR make all of the changes. Should look a lot like the try PR you already did, just with some expansions.